skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkikoUIView.kt

```diff
@@ -181,13 +182,26 @@ class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol,
         return _pointInside(skiaPoint, withEvent)
     }
 
+    private var touchesCount = 0
+        set(value) {
+            field = value
+
+            skiaLayer?.redrawer?.needsProactiveDisplayLink = value != 0
```

Comment:
A few suggestions to improve readability:
1. Move logic from setter to separate call to be more explicit
2. Wrap `value != 0` to separate getter
3. Add a comment with explanation why touches affect display link